lib/services/transaction-manager.js

```diff
@@ -14,10 +14,12 @@ export default class TransactionManager {
   }
 
   _getTransactionKey (productId) {
-    return `${productId}-${+new Date()}`
+    if (productId)
+      return `${productId}-${+new Date()}`
+    return `${+new Date()}`
```

Comment:
You're right. I kept `productId` because of some tests, but now it's removed and I refactored the tests: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/23/commits/47b3967b8702fffe39e8735e4401cd39e6d15390